tests/2015-08-01/Microsoft.Compute.tests.json

```diff
@@ -13,6 +13,96 @@
     },
 
     {
+      "name": "availabilitySets - happy test platformFaultDomainCount and platformUpdateDomainCount in range",
+      "definition": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets",
+      "json": {
+        "type": "Microsoft.Compute/availabilitySets",
+        "apiVersion": "2015-05-01-preview",
+        "location": "West Us",
+        "properties": {
+          "platformUpdateDomainCount": 10,
+          "platformFaultDomainCount": 2
+        }
+      }
+    },
+
+    {
+      "name": "availabilitySets - UpdateDomainCount out of range - less",
+      "expectedErrors": [
+        {
+          "message": "Data does not match any schemas from \"oneOf",
+          "dataPath": "/properties/platformUpdateDomainCount"
+        }
+      ],
+      "definition": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets",
+      "json": {
+        "type": "Microsoft.Compute/availabilitySets",
+        "apiVersion": "2015-05-01-preview",
+        "location": "West Us",
+        "properties": {
+          "platformUpdateDomainCount": 0
+        }
+      }
+    },
+
+    {
+      "name": "availabilitySets - UpdateDomainCount out of range - more",
+      "expectedErrors": [
+        {
+          "message": "Data does not match any schemas from \"oneOf",
+          "dataPath": "/properties/platformUpdateDomainCount"
+        }
+      ],
+      "definition": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets",
+      "json": {
+        "type": "Microsoft.Compute/availabilitySets",
+        "apiVersion": "2015-05-01-preview",
+        "location": "West Us",
+        "properties": {
+          "platformUpdateDomainCount": 25
+        }
+      }
+    },
+
+    {
+      "name": "availabilitySets - FaultDomainCount out of range - less",
+      "expectedErrors": [
+        {
+          "message": "Data does not match any schemas from \"oneOf",
+          "dataPath": "/properties/platformFaultDomainCount"
+        }
+      ],
+      "definition": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets",
+      "json": {
+        "type": "Microsoft.Compute/availabilitySets",
+        "apiVersion": "2015-05-01-preview",
+        "location": "West Us",
+        "properties": {
+          "platformFaultDomainCount": 0
+        }
+      }
+    },
+
+    {
+      "name": "availabilitySets - FaultDomainCount out of range - more",
+      "expectedErrors": [
+        {
+          "message": "Data does not match any schemas from \"oneOf",
```

Comment:
There are tests failing now for runSchemaTests.js, mostly because the error message does not match. Please fix.

Error: The error message for error 1 was not the expected error.
Expected: "Data does not match any schemas from \"oneOf"
Actual:   "Data does not match any schemas from \"oneOf\""